src/app.ts

```diff
@@ -694,7 +694,7 @@ export const initializeAppModules = () => {
       action: (output) => {
         const mediaInputs = {
           maxMediaCount: 5,
-          mediaType: microsoftTeams.MediaType.Image
+          mediaType: 1
```

Comment:
why enum is not working here. atleast can you please create an enum in this file is use it to make it more descriptive.

Reply:
added comments to make it more descriptive.